sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/logs/_exporter.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import logging
+from typing import Sequence, Any
+
+from opentelemetry.sdk._logs import LogData
+from opentelemetry.sdk._logs.severity import SeverityNumber
+from opentelemetry.sdk._logs.export import LogExporter, LogExportResult
+from opentelemetry.sdk.util import ns_to_iso_str
+
+from azure.monitor.opentelemetry.exporter import _utils
+from azure.monitor.opentelemetry.exporter._generated.models import (
+    MessageData,
+    MonitorBase,
+    TelemetryItem
+)
+from azure.monitor.opentelemetry.exporter.export._base import (
+    BaseExporter,
+    ExportResult,
+)
+
+_logger = logging.getLogger(__name__)
+
+_DEFAULT_SPAN_ID = 0
+_DEFAULT_TRACE_ID = 0
+
+__all__ = ["AzureMonitorLogExporter"]
+
+
+class AzureMonitorLogExporter(BaseExporter, LogExporter):
+    """Azure Monitor Log exporter for OpenTelemetry."""
+
+    def export(self, batch: Sequence[LogData], **kwargs: Any) -> LogExportResult: # pylint: disable=unused-argument
+        """Export log data
+        :param logs: Open Telemetry LogData to export.
+        :type logs: Sequence[~opentelemetry._logs.LogData]
```

Comment:
docstring says logs, param name is batch (i'm assuming param name should be lgs :) )